docs/network-flow-visibility.md

```diff
@@ -23,72 +19,25 @@
         - [Service Customization](#service-customization-1)
         - [Performance Configuration](#performance-configuration)
       - [Persistent Volumes](#persistent-volumes)
```

Comment:
Again not introduced by this PR, could you help put Persistent Volumes section under ClickHouse Configuration instead of parallel to it, as this section is about creating Persistent Volumes for ClickHouse. Thanks a lot!

Reply:
Done.